### PR TITLE
Image uploading & URL source (OCT-198)

### PR DESCRIPTION
**Pull request by AshleyRedmanJisc**
The purpose of this PR was provide the ability to upload multiple files or provide an image via URL for the TextEditor of a publication.

---

### Acceptance Criteria: for [Jira ticket 198 ](https://jiscdev.atlassian.net/jira/software/projects/OCT/boards/836?selectedIssue=OCT-198)

---

### Screenshots:

Screenshot 2022-04-22 at 17 12 53
Screenshot 2022-04-22 at 17 12 36
Screenshot 2022-04-22 at 17 12 21
Screenshot 2022-04-22 at 17 12 00

### CODE REVIEW

**Comment by AshleyRedman** [comment]
Focus states need to be handled

File: ui/src/components/FileUpload/index.tsx [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+                const base64 = await Helpers.getBase64FromFile(file);
+                setPreviewBase64((prevState) => [...prevState, { name: file.name, base64 }]);
+            }
+        }
+    }, []);
+
+    const { getRootProps, getInputProps, isDragActive } = DropZone.useDropzone({ onDrop });
+
+    return (
+        <section key="file-upload" className="relative">
```
Comment (nathan-at-jisc) [inline ui/src/components/FileUpload/index.tsx:30]: Can we remove `key` from the section, or is it used?

Comment (AshleyRedmanJisc) [reply ui/src/components/FileUpload/index.tsx:30]: Applied in latest commit

**Review by nathan-at-jisc — APPROVED** [review]
LGTM

**Review by nathan-at-jisc — APPROVED** [review]

**Pull request merged**